docs/CONTRIBUTING.md

```diff
@@ -2,26 +2,39 @@ Contributing
 ============
 
 #Tests
-There are several test that need to be executed to be able to test the module:
+There are several test that need to be executed to be able to test the module. When adding a PR all tests must at least pass the local tests.
+Each PR is automatically tested against Sauce Labs, see [Travis-ci with Sauce Labs](#travis-ci-with-sauce-labs).
+Before approving a PR the core contributers will test the PR against emulators / simulators / real devices.
 
 ## Local
-- `npm t` or `npm t -- -e local`: Run all tests on a local machine on Chrome and Firefox (job uses direct connect, first run `npm run wd-update` to update the webdriver. 
+**Make sure a local webdriver is installed, see [How to run a local webdriver](#how-to-run-a-local-webdriver) and start your webdriver.** 
+(*DirectConnect from protractor itself is not stable enough to run our tests, that's why we we use a local webdriver*).
+
+First a local baseline needs to be created. This can be done with `npm run test.init`. This command will create a folder called `localBaseline` that will hold all the baseline images.
+Then run `npm run test.local`. This will run all tests on a local machine on Chrome (job uses direct connect, first run `npm run wd-update` to update the webdriver. 
 This needs to be done once after install).
 
-## local on Appium
-**First make sure Appium v 1.5.3 or higher is installed**
+### How to run a local webdriver
+If you never installed a local webdriver, follow steps 1 and 2, else start the webdriver with step 3
 
-- `npm t -- -e android.adb`: Run all tests with Appium on an Android emulator with the ADB driver on Chrome (`appium --port 4728 --avd AVD_for_Nexus_5_by_Google`)
-- `npm t -- -e android.chromedriver`: Run all tests with Appium on an Android emulator with the ChromeDriver on Chrome (`appium --port 4727 --avd AVD_for_Nexus_5_by_Google`)
-- `npm t -- -e ios.simulator`: Run all tests with Appium on Apple iOS simulator on Safari (`appium --port 4726`)
+1. install a local webdriver with `npm install webdriver-manager -g`
+2. then `webdriver-manager update` to download all the webdrivers
+3. then `webdriver-manager start`, this will start a webdriver on `localhost:4444`
 
 ## Travis-ci with Sauce Labs
-- `npm t -- -e saucelabs`: This command is used to test the build through [Travis-ci](https://travis-ci.org/wswebcreation/protractor-image-comparison/). 
+- `npm run test.ci`: This command is used to test the build through [Travis-ci](https://travis-ci.org/wswebcreation/protractor-image-comparison/). 
```

Comment:
I can't get this to work, should I be able to run this? From the output it looks like it's expecting a selenium server at localhost:4445, but nothing's there.